pkg/ipfailover/keepalived/generator.go

```diff
@@ -39,8 +36,8 @@ func generateEnvEntries(name string, options *ipfailover.IPFailoverConfigCmdOpti
 		"OPENSHIFT_HA_PREEMPTION":        options.Preemption,
 		"OPENSHIFT_HA_CHECK_INTERVAL":    interval,
 		// "OPENSHIFT_HA_UNICAST_PEERS":     "127.0.0.1",
-	})
-	return env
+	}
+	return app.NewEnvironmentFromMap(env)
```

Comment:
my inclination would be to factor out usage of app.Environment from outside its home territory.